xray/plot/plot.py

```diff
@@ -211,7 +211,8 @@ def _update_axes_limits(ax, xincrease, yincrease):
 
 def _determine_cmap_params(plot_data, vmin=None, vmax=None, cmap=None,
                            center=None, robust=False, extend=None,
-                           levels=None, filled=True, cnorm=None):
+                           levels=None, filled=True, cnorm=None,
+                           robust_percentile=2):
```

Comment:
do we really need both the `robust` and `robust_percetile` keyword arguments.  Maybe just allow `robust` to be `None`, `True`, `False`, or an integer specifying the percentile (default is None / False and True defaults to 2).